### PR TITLE
update browser panel description

### LINKED ISSUE #637: postgres provider: allow copying tables from one schema to an other
https://github.com/qgis/QGIS/commit/fd2ad1329ae2238d64287fb527a7043bc5f73cf7
Author: Juergen E. Fischer
Date:   Fri Sep 11 13:38:18 2015 +0200

### LINKED ISSUE #713: [FEATURE] Allow hiding paths from the browser panel
Original commit: https://github.com/qgis/QGIS/commit/d021100e5ec994fc1750df50a6a3d5776cee2502 by NathanW2

Unfortunately this lazy coder did not write a description... :-(

### LINKED ISSUE #714: [FEATURE] Add project loading support from browser
Original commit: https://github.com/qgis/QGIS/commit/ffd7c9436fb2888b8e5ada52968d231f49173876 by NathanW2

Unfortunately this lazy coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by DelazJ**
fixes #714 
fixes #713 
also fixes #637
Add more description, replaces |qg| and removes some step-by-step description that were not really true nor well placed imho (comments inline)

### CODE REVIEW

File: source/docs/user_manual/qgis_browser/qgis_browser.rst [hunk 1]
```diff
@@ -59,33 +79,26 @@ This is handy when you need to move files or layers between locations.
...
 
-   |qg| Browser panels side-by-side
+   QGIS Browser panels side-by-side
 
-|qg| automatically looks for the coordinate reference system (CRS) and zooms to
-the layer extent if you work in a blank |qg| project. If there are already
-files in your project, the file will just be added, and in the case that it has the same
-extent and CRS, it will be visualized. If the file has another CRS and layer
-extent, you must first right-click on the layer and choose
-:guilabel:`Set Project CRS from Layer`. Then choose :guilabel:`Zoom to Layer Extent`.
```
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: This should be in first chapters (gui?) though I wonder if projection setting advice is a good one (applicable in all situations)

File: source/docs/user_manual/qgis_browser/qgis_browser.rst [hunk 2]
```diff
@@ -2,53 +2,73 @@
...
-This is handy when you need to move files or layers between locations.
-
-#. Activate a second |qg| Browser: Right-click on the toolbar and check
-   |checkbox|:guilabel:`Browser (2)`, or select it from
-   :menuselection:`Settings --> Panels`.
-#. Drag the panel into the legend window.
-#. Navigate to the :guilabel:`Browser (2)` tab and browse for a shapefile in
-   your file system.
-#. Select a file with the left mouse button. Now you can use the
-   |mActionAdd|:sup:`Add Selected Layers` icon to add it into the current project.
```
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: One would expect to have the last step being a drag-and-drop within folders, not an adding to map canvas, already described in the previous (and now removed) ste-by-step

File: source/docs/user_manual/qgis_browser/qgis_browser.rst [hunk 3]
```diff
@@ -2,53 +2,73 @@
...
-#. Click on the :guilabel:`Browser` tab.
-#. Browse in your filesystem and choose the :file:`shapefile` folder from
-   :file:`qgis_sample_data` directory.
-#. Press the :kbd:`Shift` key and select the :file:`airports.shp` and
-   :file:`alaska.shp` files.
-#. Press the left mouse button, then drag and drop the files into the map canvas.
-#. Right-click on a layer and choose :guilabel:`Set project CRS from layer`.
-   For more information see :ref:`label_projections`.
-#. Click on |mActionZoomFullExtent| :sup:`Zoom Full` to make the layers
-   visible.
```
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: This could rather be in the first chapter describing the interface.

**Comment by DelazJ** [comment]
also fixes #637

File: source/docs/user_manual/qgis_browser/qgis_browser.rst [hunk 4]
```diff
@@ -2,53 +2,72 @@
...
+  by selecting **Add Layer** or **Add selected layers** in the context menu.
+* |mActionDraw| Refresh the browser tree
+* |mActionFilterMap| search for specific data. Enter a search word or wildcard
+  and the browser will filter the tree to only show paths to matching DB tables, filenames
+  or folders -- other data or folders won't be displayed. See the Browser Panel(2)
+  example on the figure_browser_panels_. The comparison can be case-sensitive or not.
+  It can also be set to:
+
+  * normal: return any item containing the search text
+  * using a joker: fine tune the search using ``?`` and/or ``*`` characters to
```
Comment (yjacolin) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: joker -> wildcard

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 3 above)
Comment (yjacolin) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: browser panel is not enabled by default, isn't it?

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 1 above)
Comment (yjacolin) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: Same comment, gui chapter is for default tools.

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 4 above)
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: indeed :)

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 3 above)
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: Indeed, it isn't enabled by default. Why are u asking? Is it about your above comment on gui chapter?

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 1 above)
Comment (yjacolin) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: @DelazJ what about this, do your comment is just a though or do you want more comment?

I am ok to merge it as is anyway.

**Comment by yjacolin** [comment]
> Indeed, it isn't enabled by default. Why are u asking? Is it about your above comment on gui chapter?

Yes, I think that the gui chapter is to describe default tools and then each chapter gives more information about "custom" options.

File: source/docs/user_manual/qgis_browser/qgis_browser.rst (hunk 1 above)
Comment (DelazJ) [inline source/docs/user_manual/qgis_browser/qgis_browser.rst]: just to explain why I removed this large part of the chapter!

**Comment by DelazJ** [comment]
ok

**Pull request merged**